psydac/mapping/tests/test_discrete_mapping.py

```diff
@@ -267,42 +267,42 @@ def test_parallel_jacobians_irregular(geometry, npts_irregular):
         os.remove('result_parallel.h5')
 
 
-def test_nurbs_circle():
-    rmin, rmax = 0.2, 1
-    c1, c2 = 0, 0
+# def test_nurbs_circle():
```

Comment:
you can mark this test as failing using pytest without the need to comment it

Reply:
indeed, I missed this one 👍